exercises/concept/elyses-enchantments/.docs/hints.md

```diff
@@ -0,0 +1,35 @@
+# Hints
+
+## 1. Retrieve a card from a stack
+
+- `Vector` indices start at `1`.
+
+## 2. Exchange a card in the stack
+
+- `Vector`s are mutable, you can change their contents at any time.
```

Comment:
### Previous discussion

> Maybe it would be better to do this without mutation first?

_Originally posted by @cmcaine in https://github.com/exercism/v3/pull/3080#r563385512_

> Do you mean we should have a copying method first?
> 
> I can't think of many things to do with arrays other than indexing (and maths) that don't require mutation.

_Originally posted by @SaschaMann in https://github.com/exercism/v3/pull/3080#r563393209_

> I mean we could show how to do all these things with copying (possibly even with tuples instead of arrays?).

_Originally posted by @cmcaine in https://github.com/exercism/v3/pull/3080#r563459072_

Reply:
I don't think this exercise works well with tuples, since it relies on mutation quite a bit. Maybe we could have a similar one with tuples?

A lot of these basic methods to manipulate arrays don't even have copying equivalents, e.g. there's no `push` or `deleteat!`. Perhaps we could mention in the text that, if copying is required, one can use `f!(copy(x))` instead?